rootless-install.sh

```diff
@@ -114,7 +114,13 @@ checks() {
 	if [ -x "$BIN/$DAEMON" ]; then
 		# If rootless installation is detected print out the modified PATH and DOCKER_HOST that needs to be set.
 		echo "# Existing rootless Docker detected at $BIN/$DAEMON"
-		echo "# See https://docs.docker.com/go/rootless/ for the usage."
+		echo
+		echo "# To reinstall or upgrade rootless Docker, please run the following commands and then rerun the installation script:"
```

Comment:
Trying to avoid "please" in this sentence to make it a bit more "to the point"; 

```suggestion
		echo "# To reinstall or upgrade rootless Docker, run the following commands and then rerun the installation script:"
```